sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/LeaseRenewerImpl.java

```diff
@@ -63,7 +63,11 @@ public LeaseRenewerImpl(Lease lease, LeaseManager leaseManager, Duration leaseRe
             })
             .then()
             .onErrorResume(throwable -> {
-                logger.error("Partition {}: renew lease loop failed.", this.lease.getLeaseToken(), throwable);
+                if (throwable instanceof LeaseLostException) {
+                    logger.warn("Partition {}: renew lease loop failed.", this.lease.getLeaseToken(), throwable);
```

Comment:
let's change these to debug, or error if you want to. But not warn.

Reply:
Fixed